### PR TITLE
Video 4454 message list scrolling

### PR DESCRIPTION
**Pull request by olipyskoty**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [X] I acknowledge that all my contributions will be made under the project's license.

### JIRA link(s):

- [VIDEO-4454](https://issues.corp.twilio.com/browse/VIDEO-4454)

### Description

This PR is for a new component that is a scrollable container to be wrapped around the `<MessageList />` component. The`<MessageList />` will ultimately grow taller than its container as it continues to receive new messages, and users will need to have the ability to scroll up and down the chat window. A "new message" button will be displayed if the user receives a new message and is not scrolled to the bottom. The button will disappear if the user clicks on the button, or manually scrolls to the bottom. Otherwise, the chat window will auto-scroll to the bottom if a new message is received, and the user is already scrolled to the bottom.

I've included some comments in the code, but let me know if any of it is overkill, or if I should add more. 

**Screenshot:**

### Before review
* [ ] Updated CHANGELOG.md if necessary
* [ ] Added unit tests if necessary
* [ ] Updated affected documentation
* [x] Verified locally with `npm test`
* [x] Manually sanity tested running locally
* [x] Included screenshot as PR comment (if needed)
* [x] Ready for review

### Before merge
* [ ] Got one or more +1s
* [ ] Re-tested if necessary

### CODE REVIEW

File: src/components/ChatWindow/MessageList/MessageListScrollContainer/MessageListScrollContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  showButton: boolean;
+  messageNotificationCount: number;
+}
+
+/*
+ * This component is a container for the MessageList component. It enables scrolling so that users
+ * can scroll through the messages as the MessageList grows taller than this container.
+ *
+ * Note that this component is tested with Cyprus only.
+ */
```
Comment (timmydoza) [inline src/components/ChatWindow/MessageList/MessageListScrollContainer/MessageListScrollContainer.tsx:56]: I think the PR is good to go, but I think these comments could be a little more descriptive. I think the PR description is a good example of what could go here.

**Review by timmydoza — APPROVED** [review]

File: src/components/ChatWindow/MessageList/MessageListScrollContainer/MessageListScrollContainer.tsx (hunk 1 above)
Comment (olipyskoty) [reply src/components/ChatWindow/MessageList/MessageListScrollContainer/MessageListScrollContainer.tsx:56]: thank you! i also spelled "Cypress" wrong 🙃

File: src/components/ChatWindow/MessageList/MessageListScrollContainer/MessageListScrollContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+ * This component is a scrollable container that wraps around the 'MessageList' component.
+ * The MessageList will ultimately grow taller than its container as it continues to receive
+ * new messages, and users will need to have the ability to scroll up and down the chat window.
+ * A "new message" button will be displayed when the user receives a new message, and is not scrolled
+ * to the bottom. This button will be hidden if the user clicks on it, or manually scrolls
+ * to the bottom. Otherwise, this component will auto-scroll to the bottom when a new message is
+ * received, and the user is already scrolled to the bottom.
+ *
+ * Note that this component is tested with Cypress only.
+ */
```
Comment (olipyskoty) [inline src/components/ChatWindow/MessageList/MessageListScrollContainer/MessageListScrollContainer.tsx:61]: @timmydoza do you mind proofreading this? want to make sure it's not **too** big

Comment (timmydoza) [reply src/components/ChatWindow/MessageList/MessageListScrollContainer/MessageListScrollContainer.tsx:61]: Looks good! Not too big. Can't have too many comments 🙂

**Pull request merged**